File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/ButtonsColorFragment.java
package com.regula.facesample.items.customization.fragment;

import android.graphics.Color;

import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 6.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class ButtonsColorFragment extends FaceDefaultUiFragment {

    @Override
    public int getButtonsColor(boolean isLightOn) {
        return isLightOn ? Color.RED : Color.GREEN;
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/fragments/ButtonsColorFragment.kt
package com.regula.facesamplekotlin.items.fragments

import android.graphics.Color
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class ButtonsColorFragment: FaceDefaultUiFragment() {

    override fun getButtonsColor(isLightOn: Boolean): Int {
        return if (isLightOn) Color.RED else Color.GREEN
    }
}